packages/node_modules/@webex/plugin-meetings/src/networkQualityMonitor/index.js

```diff
@@ -0,0 +1,178 @@
+import EventsScope from '../common/events/events-scope';
+import {EVENT_TRIGGERS, STATS} from '../constants';
+
+
+/**
+  * Meeting - network quality judgment event
+  * Emitted on each interval of retrieving stats Analyzer data
+  * @event network:quality:judgment
+  * @type {Object}
+  * @property {string} mediaType {video|audio}
+  * @property {boolean} networkAcceptable is network overal acceptable
+  * @property {Object} networkQualityDataStructure for each indicator type
+  * @property {boolean} networkQualityDataStructure.acceptable
+  * @property {Number} networkQualityDataStructure.value - score or RTC value for each indicator type
+  * @memberof NetworkQualityMonitor
+  */
+/**
+ * NetworkQualityMonitor class that will emit events based on detected quality
+ *
+ * @class NetworkQualityMonitor
+ * @extends {EventsScope}
+ */
+export default class NetworkQualityMonitor extends EventsScope {
+  /**
+   * Creates a new instance of NetworkQualityMonitor
+   * @constructor
+   * @public
+   * @param {Object} config
+   */
+  constructor(config) {
+    super();
+    this.config = config;
+    this.indicatorTypes = Object.freeze({
+      PACKETLOSS: 'packetLoss',
+      LATENCY: 'latency',
+      JITTER: 'jitter'
+    });
+    this.adequeteExistingNetworkQuality = true;
```

Comment:
what does the adequeteExistingNetworkQuality, is it to determine if the values has changed ,

Can we rename it to something else and provide some jsdoc for these values

Reply:
adequate just means good enough not necessarily good - can change term to whatever you like and I will also update the class to include js doc for class properties